### PR TITLE
Add missing variable and upload of rpm to github

### PR DESCRIPTION
**Pull request by afbjorklund**

### CODE REVIEW

File: hack/jenkins/release_github_page.sh [hunk 1]
```diff
@@ -121,6 +122,7 @@ FILES_TO_UPLOAD=(
     'minikube-windows-amd64.sha256'
```
Comment (tstromberg) [inline hack/jenkins/release_github_page.sh]: While you are here, I'm pretty sure this should be minikube-windows-amd64.exe.sha256

Comment (afbjorklund) [reply hack/jenkins/release_github_page.sh]: Yes, that looks right.

**Comment by afbjorklund** [comment]
For the `v0.35.0`, I just removed the rpm link from the release text.

File: hack/jenkins/release_github_page.sh (hunk 1 above)
Comment (afbjorklund) [reply hack/jenkins/release_github_page.sh]: Added https://github.com/kubernetes/minikube/pull/3838

**Pull request merged**